pkg/api/events.go

```diff
@@ -47,7 +49,7 @@ func (p *v1Provider) ListEvents(res http.ResponseWriter, req *http.Request) {
 	if offsetStr != "" {
 		parsedOffset, err := strconv.ParseUint(offsetStr, 10, 32)
 		if err != nil {
-			http.Error(res, "Invalid offset value", http.StatusBadRequest)
+			ValidationError(res, fmt.Errorf("invalid offset value: %w", err))
```

Comment:
What is the reason for this change? I would have thought that `http.Error` is a more canonical way of doing it?

I am not a big fan of error-wrappers so I guess I am a bit biased here.

Reply:
I agree, I was trying to match some of our other apis and how they do it for consistency. But I'm likely being too dogmatic there with an end goal, without much logic towards it outside of consistency. 

I'll look at it again. I also need to do another run through comparing the setup to our reference architecture for golang apis, and make sure I'm matching that as well.

Reply:
Agreed, removed the wrapper.